pom.xml

```
@@ -357,6 +357,34 @@
     </dependency>
 
   </dependencies>
+  <dependencyManagement>
+    <dependencies>
+      <!-- Align ALL Spring 5.3.x artifacts -->
+      <dependency>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-framework-bom</artifactId>
+        <version>5.3.39</version>
```

we already define the global spring verison as a property in the base pom. let's keep them aligned:
```suggestion
        <version>${spring.version}</version>
```